projects/tuist/fixtures/app_with_spm_dependencies/Tuist/Dependencies.swift

```diff
@@ -4,13 +4,14 @@ import ProjectDescriptionHelpers
 let packages: [Package] = [
     .package(url: "https://github.com/Alamofire/Alamofire", .upToNextMajor(from: "5.6.0")),
     .package(url: "https://github.com/facebook/facebook-ios-sdk", .upToNextMajor(from: "13.2.0")),
-    .package(url: "https://github.com/firebase/firebase-ios-sdk", .upToNextMajor(from: "9.3.0")),
+    .package(url: "https://github.com/firebase/firebase-ios-sdk", .upToNextMajor(from: "10.12.0")),
     .package(url: "https://github.com/pointfreeco/swift-composable-architecture", .upToNextMinor(from: "0.40.0")),
     .package(url: "https://github.com/iterable/swift-sdk", .upToNextMajor(from: "6.4.0")),
     .package(url: "https://github.com/Trendyol/ios-components", .revision("c9260bfe203a16a278eca5542c98455eece98aa4")),
-    .package(url: "https://github.com/stripe/stripe-ios", .upToNextMajor(from: "22.4.0")),
+    .package(url: "https://github.com/stripe/stripe-ios", .upToNextMajor(from: "23.11.1")),
```

Comment:
Why these changes? Let's try to keep changes related to this PR only

Reply:
Old version firebase won't compile with Xcode 15, and stripe has some shared dependency with firebase, has to update as well